app/templates/-current-user-header.hbs

```diff
@@ -6,34 +6,18 @@
 
     <nav>
       <ul class="nav nav-pills navbar-right user-nav">
-        <li>
-          {{#if model.organization}}
-            {{link-to model.organization.name "index"}}
-          {{else}}
-            {{link-to model.stack.organization.name "index"}}
-          {{/if}}
-        </li>
-
         <li class="dropdown current-user">
           {{#dashboard-dropdown title=session.currentUser.name defaultPath="index"}}
 
-            {{#dashboard-dropdown-item}}
-              {{link-to "Dashboard" "index"}}
-            {{/dashboard-dropdown-item}}
-
-            {{#dashboard-dropdown-item}}
-              {{#link-to "settings"}}Settings{{/link-to}}
-            {{/dashboard-dropdown-item}}
+            {{#each session.currentUser.organizations as |organization|}}
```

Comment:
This is where I might expect that organizations (already fetch by the route) is referenced.